src/sync.cpp

```diff
@@ -57,7 +57,7 @@ struct CLockLocation {
 
     std::string ToString() const
     {
-        return tfm::format(
+        return strprintf(
```

Comment:
IIUC this change is only needed due to the linter change, e.g. causing `./test/lint/lint-format-strings.sh` to raise
```bash
src/sync.cpp: Expected 0 argument(s) after format string but found 4 argument(s): tfm::format( "%s %s:%s%s (in thread %s)", mutexName, sourceFile, itostr(sourceLine), (fTry ? " (TRY)" : ""), m_thread_name)
```
while on master it does not raise.

If that is correct (or if I'm misunderstanding) perhaps an quick note in the commit and PR description could be helpful to other people.

Reply:
Correct, but I didn't rewrite the commits to preserve the acks.